feature/quicksettings/src/main/java/com/google/jetpackcamera/feature/quicksettings/ui/QuickSettingsComponents.kt

```diff
@@ -77,7 +77,7 @@ fun ExpandedQuickSetRatio(setRatio: (aspectRatio: AspectRatio) -> Unit, currentR
             },
             {
                 QuickSetRatio(
-                    modifier = Modifier.testTag("QuickSetAspectRatio1:1"),
+                    modifier = Modifier.testTag("QuickSettingsAspectRatio1:1Button"),
```

Comment:
Use the constants here rather than string literals.

Reply:
TestTags are in app module, using it here would cause circular dependency